client/src/modules/Admin/Components/Admin.tsx

```diff
@@ -450,7 +451,7 @@ export const Admin = () => {
 
   function adminLogin() {
     if (loading) {
-      return <div>Loading...</div>
+      return <div><Loading /></div>
```

Comment:
I think you can just directly return one the Loading component, not being wrapped in div tags for cleanliness.